edge-modules/load-gen/Program.cs

```diff
@@ -8,15 +8,17 @@ namespace LoadGen
     using Microsoft.Azure.Devices.Client;
     using Microsoft.Azure.Devices.Edge.ModuleUtil;
     using Microsoft.Azure.Devices.Edge.Util;
-    using Microsoft.Azure.Devices.Shared;
     using Microsoft.Extensions.Logging;
     using Newtonsoft.Json;
+    using Message = Microsoft.Azure.Devices.Client.Message;
 
     class Program
     {
         static readonly ILogger Logger = ModuleUtil.CreateLogger("LoadGen");
-
-        static long messageIdCounter = 0;
+        static readonly Guid BatchId = Guid.NewGuid();
+        static readonly string DeviceId = Environment.GetEnvironmentVariable("IOTEDGE_DEVICEID");
+        static readonly string ModuleId = Environment.GetEnvironmentVariable("IOTEDGE_MODULEID");
```

Comment:
This should have been in Settings.cs, but irrelevant since I am reverting all changes to this module.